src/routes/user/userRoute.js

```diff
@@ -0,0 +1,13 @@
+const { createUser, userLogin } = require('../../controllers/user.controllers');
```

Comment:
Please consider changing all the `require` imports to `import` imports and this import path to an absolute one.